blueman/gui/GenericList.py

```diff
@@ -29,15 +29,15 @@ def _load(self, data):
 
             self.ids[data[i][0]] = i
 
-            if len(data[i]) == 5 or len(data[i]) == 6:
+            if len(data[i]) == 4 or len(data[i]) == 5:
 
-                column = Gtk.TreeViewColumn(data[i][4])
+                column = Gtk.TreeViewColumn(data[i][3])
```

Comment:
Instead of what seems to have been `None` in all cases you're now passing the "renderer_attributes" `dict` as what should be a `Gtk.CellArea`. I wonder how does not cause an error.